brayns/common/scene/Model.h

```diff
@@ -268,6 +290,12 @@ class Model
     Boxf _bounds;
     bool _useSimulationModel{false};
 
+    std::map<size_t, std::vector<uint32_t>> _SDFGeometryIndices;
+    std::vector<size_t> _SDFNeighboursFlat;
+    std::vector<std::vector<size_t>> _SDFNeighbours;
+    std::vector<SDFGeometry> _SDFGeometries;
+    bool _SDFGeometriesDirty{false};
```

Comment:
Not sure, but that's a lot of additions to the Model class, could maybe go into a separate SDF class instanced here..?

Reply:
I was wondering in general if this should be in the core at all or rather be a 'real' plugin? Maybe not now, but latest when the circuit loader and friends are moved to a plugin, this shall move as well.